src/app.mjs

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-console */
+import readline from 'readline';
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function getNumbersForGame() {
+  let i = '';
+
+  while (i.length < 4) {
+    const newNumber = Math.floor(Math.random() * 10);
+
+    if (!i.includes(newNumber)) {
+      i += newNumber;
+    }
+  }
+
+  return i.split('');
+}
+
+const randomNumberFind = getNumbersForGame();
+
+console.log(randomNumberFind);
+
+function calculation(arrayNumbers) {
+  let cows = 0;
+  let bulls = 0;
+
+  const alreadySearched = [];
+
+  let iterator = 0;
+
+  while (iterator < 4) {
+    if (arrayNumbers[iterator] === randomNumberFind[iterator]) {
```

Comment:
`arrayNumbers[iterator]` can we move it separate variable, because I see 6 reuse